docs/rest/openapi_demo.md

```diff
@@ -142,6 +142,75 @@ For example, to create a "sidetree" public DID in alice agent, go to `HTTP POST
+    ```
+    The response should be the following:
+    ```json
+    {}
+    ```
+3. Accept a presentation (Alice).
+
+    To accept a presentation you need to know `PIID`.
+    You can achieve that by performing `/presentproof/actions` API call.
+    ```
+    curl -X GET "https://localhost:9082/presentproof/actions" -H  "accept: application/json"
+    ```
+    The response should be similar to the following:
+    ```json
+    {"actions":[{"PIID":"80f8b418-4818-4af6-8915-f299b974f5c2","Msg":{"@id":"bad7cffc-ee7f-4755-8b20-7d70104c4034","@type":"https://didcomm.org/present-proof/2.0/presentation","presentations~attach":[{"data":{"base64":"ZXlKaGJHY2lPaUp1YjI1bElpd2lkSGx3SWpvaVNsZFVJbjAuZXlKcGMzTWlPaUprYVdRNlpYaGhiWEJzWlRwbFltWmxZakZtTnpFeVpXSmpObVl4WXpJM05tVXhNbVZqTWpFaUxDSnFkR2tpT2lKMWNtNDZkWFZwWkRvek9UYzRNelEwWmkwNE5UazJMVFJqTTJFdFlUazNPQzA0Wm1OaFltRXpPVEF6WXpVaUxDSjJjQ0k2ZXlKQVkyOXVkR1Y0ZENJNld5Sm9kSFJ3Y3pvdkwzZDNkeTUzTXk1dmNtY3ZNakF4T0M5amNtVmtaVzUwYVdGc2N5OTJNU0lzSW1oMGRIQnpPaTh2ZDNkM0xuY3pMbTl5Wnk4eU1ERTRMMk55WldSbGJuUnBZV3h6TDJWNFlXMXdiR1Z6TDNZeElsMHNJbWh2YkdSbGNpSTZJbVJwWkRwbGVHRnRjR3hsT21WaVptVmlNV1kzTVRKbFltTTJaakZqTWpjMlpURXlaV015TVNJc0ltbGtJam9pZFhKdU9uVjFhV1E2TXprM09ETTBOR1l0T0RVNU5pMDBZek5oTFdFNU56Z3RPR1pqWVdKaE16a3dNMk0xSWl3aWRIbHdaU0k2V3lKV1pYSnBabWxoWW14bFVISmxjMlZ1ZEdGMGFXOXVJaXdpUTNKbFpHVnVkR2xoYkUxaGJtRm5aWEpRY21WelpXNTBZWFJwYjI0aVhTd2lkbVZ5YVdacFlXSnNaVU55WldSbGJuUnBZV3dpT201MWJHeDlmUS4="}}],"~thread":{"thid":"80f8b418-4818-4af6-8915-f299b974f5c2"}},"MyDID":"did:peer:1zQmRmFgG6PUX8d9Zrrkdh9akh3uijnoTMfx8XiRhMDSFJwT","TheirDID":"did:peer:1zQmQSLFWySB3LACeSrUpvM48QN9frMayNHypnsQjk4GhQKG"},{"PIID":"824577e2-94df-4e77-9122-3d72238a65b8","Msg":{"@id":"b0c86f29-050a-4602-8914-338b24560e2e","@type":"https://didcomm.org/present-proof/2.0/presentation","comment":"string","formats":[{"attach_id":"string","format":"string"}],"presentations~attach":[{"@id":"string","data":{"base64":"string","json":{},"links":["string"],"sha256":"string"},"description":"string","filename":"string","lastmod_time":"2020-09-17T09:21:07.243Z","mime-type":"string"}],"~thread":{"thid":"824577e2-94df-4e77-9122-3d72238a65b8"}},"MyDID":"did:peer:1zQmRmFgG6PUX8d9Zrrkdh9akh3uijnoTMfx8XiRhMDSFJwT","TheirDID":"did:peer:1zQmQSLFWySB3LACeSrUpvM48QN9frMayNHypnsQjk4GhQKG"}]}
```

Comment:
It would be nice to show the decoded Presentation Exchange payloads in the text to give the reader more context.